src/main/resources/lang/pylonbase/en.yml

```
@@ -339,3 +349,4 @@ message:
       1: "you find that a seemingly living gelatinous substance has formed in the pack"
       2: "a strange, glowing liquid spills out"
       3: "full of anticipation, you are disappointed to find that nothing happens"
+  immobilizer_on_cooldown: "<red>The immobilizer is on cooldown!"
```

```suggestion
  immobilizer_on_cooldown: "<red>The immobilizer is on cooldown"
```
Never really liked the excessive use of exclamation points in SF